### PR TITLE
Feature/seab 817/galaxy support

### PR DESCRIPTION
**Pull request by NatalieEO**
Part of https://ucsc-cgl.atlassian.net/browse/SEAB-817

### CODE REVIEW

File: install_bootstrap [hunk 1]
```diff
@@ -202,6 +235,10 @@ while [[ "${run_dockstore_launcher^^}" != 'Y' &&  "${run_dockstore_launcher^^}"
     if [ "${run_dockstore_launcher^^}" = 'Y' ] ; then
 	production='production'
 	ask_question "Are you deploying on production (true or false)?" "$PRODUCTION" "Dockstore version" $production
+
+        galaxy_plugin_version='galaxy_plugin_version'
```
Comment (denis-yuen) [inline install_bootstrap]: Super nit-picky, but Indentation seems off

File: install_bootstrap [hunk 2]
```diff
@@ -109,6 +110,38 @@ function ask_question()
...
+    prev_val=$2
+    answer=''
+    if [[ -n $prev_val ]] ; then
+        read -ep "${question_str} Previous value: "$'\n' -i "$prev_val" answer
+    else
+        read -ep "$question_str"$'\n' answer
+    fi
+
+    if [[ -z ${answer// /} ]] ; then
+        answer="NONE"
```
Comment (denis-yuen) [inline install_bootstrap]: Rather than a special value, I wonder if a null value would make more sense here.

**Comment by NatalieEO** [comment]
> Fixed typo, seems reasonable.
> One question, how does the new optional code react to the existing value on staging/prod when the value is completely missing?

If you leave the value as empty in in the compose.config and then run install_bootstrap, it'll try to download but fail. Will fix and rewrite to set to a null value instead of special one.

File: dockstore_launcher_config/compose.config [hunk 3]
```diff
@@ -29,5 +29,5 @@
 "GITHUB_APP_NAME":"foobar",
 "DOCUMENTATION_URL":"foobar",
 "FEATURED_CONTENT_URL":"foobar",
-"GALAXY_PLUGIN_VERSION":"NONE"
+"GALAXY_PLUGIN_VERSION":""
```
Comment (denis-yuen) [inline dockstore_launcher_config/compose.config]: Was more thinking `null` default would be `"GALAXY_PLUGIN_VERSION": null`
https://stackoverflow.com/questions/21120999/representing-null-in-json#21121267
I don't feel strongly about it though

**Pull request merged**